xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/mentions/mention.vm

```diff
@@ -41,6 +41,9 @@
     <td class="description">$summary</td>
     #insertEventDate($event)
 </tr>
+    <tr>
```

Comment:
since the quote might be null I guess it would need a if to not display this line at all if it's the case